### ISSUE TITLE
more vector drivers request!

### ISSUE THREAD

**Issue by qjw2bqn** [MaxRev-Dev/gdal.netcore#8]
More and more the database is used to spatial database stored ,eg PostGis,Mysql,Mongodb etc..

The Gdal.org have support about 90+ ogr drivers  [https://gdal.org/drivers/vector/index.html]

But this package only support about 75+ ogr drivers. More seriously，the usually database types doesn't be supported.

do you have plan to support postgis or other database types driver?

**Comment by MaxRev-Dev** [comment]
TLDR: No.

Why? Because, I don't have too much time to support other drivers. I've created these packages only to bring a read/write support for geospatial files. That's why I'm calling them - 'Minimals'.

But, If the community want to contribute to these repo, I will be glad to help. 

Currently, the **main problem** is with batch files for Windows. **Each** new driver must be configured via batch manually. So we must to port them to **make/cmake**. 

Of course, If you want you can try to build a version with all drivers. But again, It took a several weeks for me to bring to you a current version of packages. Not everything builds easy. Yet, I didn't figure out how to compile libtiff+libgeotiff for Windows, they are just failing on linking in gdal.
